database/migrations/2018_01_01_000000_create_laravel_translation_languages_table.php

```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class CreateLanguagesTable extends Migration
+class CreateLaravelTranslationLanguagesTable extends Migration
```

Comment:
Why do you need that name change? That will force old users to delete migration record from database. That will break backward compatibility

Reply:
It breaks my code, so it’s pretty likely that someone else with `languages` table will have the same problem.

Reply:
I get that and I'm sure others will be affected, but I don't really want to change it at this point.

Is it possible you can rename your migration?

I'm happy to merge everything else.

Reply:
You should add some info about this problem to readme then, so that people don't get confused. I will revert the name change soon.